build/BUILD.bazel

```diff
@@ -37,14 +37,25 @@ install_lualib_deps_cmd = "\n".join([
 ])
 
 wasmx_deps = select({
-    "@kong//:wasmx_flag": [
+    "@kong//:static_wasmx_flag": [
+        "@ngx_wasm_module//:lua_libs",
+    ],
+    "@kong//:dyn_wasmx_flag": [
         "@ngx_wasm_module//:lua_libs",
     ],
     "//conditions:default": [],
 })
 
 wasmx_lua_libs_cmd = select({
-    "@kong//:wasmx_flag": """
+    "@kong//:static_wasmx_flag": """
+    for fname in $(locations @ngx_wasm_module//:lua_libs); do
```

Comment:
I wonder if it's possible to de-dupe this shell command string so that it can be referenced only once for each flag in the `select()` call.

Can you give this a shot and see if it works? Honestly it's not that much prettier, but w/ the state of bazel I'll take any kind of win we can get.

```python
install_wasm_lua_libs_cmd = """
    for fname in $(locations @ngx_wasm_module//:lua_libs); do
        base=${fname##*/ngx_wasm_module/lib/}
        dest="${BUILD_DESTDIR}/openresty/lualib/$base"
        mkdir -p "$(dirname "$dest")"
        cp "$fname" "$dest"
    done
"""

wasmx_lua_libs_cmd = select({
    "@kong//:static_wasmx_flag": install_wasm_lua_libs_cmd,
    "@kong//:dyn_wasmx_flag": install_wasm_lua_libs_cmd,
    "//conditions:default": "\n",
})
```

I wouldn't consider the duplicate string a blocker though. Don't spend too much time trying to make it work if bazel fights back.

Reply:
We can also use a `config_settings_group` like https://github.com/Kong/kong/blob/master/BUILD.bazel#L231
then you can have a new condition on say `@kong//:any_wasmx_flag`